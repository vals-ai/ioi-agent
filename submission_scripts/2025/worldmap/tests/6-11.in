version https://git-lfs.github.com/spec/v1
oid sha256:f83f1c881f165c694f60886fffb1847eae63545df31e4a71718747125f95f6e7
size 720
