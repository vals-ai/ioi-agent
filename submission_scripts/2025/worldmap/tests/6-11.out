version https://git-lfs.github.com/spec/v1
oid sha256:bc564a561f67ecf3c09db569ac9442cb7bb489e32a760c924d11fd24b748719a
size 38529
